src/components/ProductionMemberModal/index.tsx

```diff
@@ -50,9 +57,14 @@ function ProductionMemberModal({
     )
     .then(res => {
       console.log(res);
-      res.data?.errors
-      ? setReqStatus({error:{ errorMessage: res.data.errors[0].message}})
-      : setReqStatus({success: "Movie created successfully :D"})
+      if(reqStatus !== null) {
+        return;
+      } else {
+        res.data?.errors
+          ? setReqStatus({error:{ errorMessage: res.data.errors[0].message}})
+          : setReqStatus({success: "producer updated successfully :D"})
```

Comment:
Should it be like "{success:`${itemToUpdate} updated successfully`});"? @Hai5edfm